src/datanode/src/tests/instance_test.rs

```diff
@@ -404,66 +379,34 @@ pub async fn test_execute_create() {
     let instance = Instance::with_mock_meta_client(&opts).await.unwrap();
     instance.start().await.unwrap();
 
-    let output = instance
-        .execute_sql(
-            r#"create table test_table(
+    let output = execute_sql(
+        &instance,
+        r#"create table test_table(
                             host string,
                             ts timestamp,
                             cpu double default 0,
                             memory double,
                             TIME INDEX (ts),
                             PRIMARY KEY(host)
                         ) engine=mito with(regions=1);"#,
-        )
-        .await
-        .unwrap();
+    )
+    .await;
     assert!(matches!(output, Output::AffectedRows(1)));
 }
 
-#[tokio::test(flavor = "multi_thread")]
-pub async fn test_create_table_illegal_timestamp_type() {
```

Comment:
Why delete this test case?

Reply:
this test case is no longer usefull, it's testing an outdated purpose, that bigint cannot be timestamp